lambdas/credential-subject/src/credential-subject-handler.ts

```diff
@@ -1,21 +1,24 @@
 import { LambdaInterface } from "@aws-lambda-powertools/commons";
-import { Logger } from "@aws-lambda-powertools/logger";
 import { UserInfoEvent } from "./user-info-event";
 import {
   BirthDate,
   CredentialSubject,
   CredentialSubjectBuilder,
   NamePart,
 } from "./credential-subject-builder";
+import { LogHelper } from "../../logging/log-helper";
+import { Context } from "aws-lambda";
 
-const logger = new Logger();
+const logHelper = new LogHelper();
```

Comment:
Since you export const { logEntry, logError } = new LogHelper();
You don't need the create the instance in the lambda. you can now use the methods directly in the lambda's do the following:

```
logEntry(context.functionName, event.govJourneyId)
logError(context.functionName, event.govJourneyId, message)
```